projects/ad738x-st/sdp-ck1z.ioc

```diff
@@ -0,0 +1,158 @@
+#MicroXplorer Configuration settings - do not modify
+CAD.formats=
+CAD.pinconfig=
+CAD.provider=
+File.Version=6
+GPIO.groupedBy=Group By Peripherals
+KeepUserPlacement=false
+Mcu.CPN=STM32F469NIH6
+Mcu.Family=STM32F4
+Mcu.IP0=NVIC
+Mcu.IP1=RCC
+Mcu.IP2=SPI1
+Mcu.IP3=SYS
+Mcu.IP4=UART5
+Mcu.IPNb=5
+Mcu.Name=STM32F469NIHx
+Mcu.Package=TFBGA216
+Mcu.Pin0=PB4
+Mcu.Pin1=PB3
+Mcu.Pin2=PC12
+Mcu.Pin3=PA15
+Mcu.Pin4=PD2
+Mcu.Pin5=PH0/OSC_IN
+Mcu.Pin6=PH1/OSC_OUT
+Mcu.Pin7=PA7
+Mcu.Pin8=VP_SYS_VS_Systick
+Mcu.PinsNb=9
+Mcu.ThirdPartyNb=0
+Mcu.UserConstants=
+Mcu.UserName=STM32F469NIHx
+MxCube.Version=6.10.0
+MxDb.Version=DB.6.0.100
```

Comment:
MxCube.Version=6.5.0
MxDb.Version=DB.6.0.50
Please use these values instead, these work on the CI and please add the builds.json file for this project so that the CI is building it and the binaries are added in the release